gallery/src/styles/NxList/NxListBulletedExample.html

```diff
@@ -6,44 +6,42 @@
     distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
 
 -->
-<div class="nx-list nx-list--bulleted">
-  <h4 class="nx-list__title">
-    List title
-  </h4>
-  <ul>
-    <li class="nx-list__item">
-      item 1
-    </li>
-    <li class="nx-list__item">
-      <span>Item 2</span>
-      <ul class="nx-list nx-list--bulleted">
-        <li class="nx-list__item">
-          item 1
-        </li>
-        <li class="nx-list__item">
-          Item 2
-        </li>
-        <li class="nx-list__item">
-          <span>item 3</span>
-          <ul class="nx-list nx-list--bulleted">
-            <li class="nx-list__item">
-              item 1
-            </li>
-            <li class="nx-list__item">
-              Item 2
-            </li>
-            <li class="nx-list__item">
-              item 3
-            </li>
-          </ul>
-        </li>
-        <li class="nx-list__item">
-          Item 4
-        </li>
-      </ul>
-    </li>
-    <li class="nx-list__item">
-      Item 3
-    </li>
-  </ul>
-</div>
+<h3 class="nx-h3">
+  Bulleted lists
+</h3>
+<ul class="nx-list nx-list--bulleted">
+  <li class="nx-list__item">
+    <span class="nx-list__text">item 1</span>
+  </li>
+  <li class="nx-list__item">
+    <span class="nx-list__text">Item 2</span>
+    <ul class="nx-list nx-list--bulleted">
+      <li class="nx-list__item">
+        <span class="nx-list__text">item 1</span>
+      </li>
+      <li class="nx-list__item">
+        <span class="nx-list__text">Item 2</span>
+      </li>
+      <li class="nx-list__item">
+        <span class="nx-list__text">item 3</span>
+        <ul class="nx-list nx-list--bulleted">
+          <li class="nx-list__item">
```

Comment:
What should happen to the list symbols beyond 3 layers deep?  Right now it just stays with squares

Reply:
> What should happen to the list symbols beyond 3 layers deep? Right now it just stays with squares

That's all that appears in Zeplin, but my feeling is that lists that go deeper than that maybe should be rethought

Reply:
@dturner32 ^^